### PR TITLE
enable ort-extensions in wasm release builds

### PR DESCRIPTION
**Pull request by guschmue**
enable ort-extensions in wasm release builds. sentence piece, gpt2, bert and word piece tokenizers for now.

wasm size will grow from 8.4MB to 8.9MB.

### CODE REVIEW

**Comment by fs-eire** [comment]
https://github.com/microsoft/onnxruntime/blob/main/tools/ci_build/github/azure-pipelines/templates/win-wasm-ci.yml#L32

this is the better place to put the build commandline

**Comment by fs-eire** [comment]
Does the current way support flag overriding?

for example, if a user want to set OCOS_ENABLE_OPENCV=ON ( I know this is not the real requirement, just for example ), then the user needs to set the "extra build flag" to `--cmake_extra_defines OCOS_ENABLE_OPENCV=ON`. The final build commandline will be `--use_extensions --cmake_extra_defines onnxruntime_WEBASSEMBLY_DEFAULT_EXTENSION_FLAGS=ON --cmake_extra_defines OCOS_ENABLE_OPENCV=ON`. will this work?

**Comment by guschmue** [comment]
the user could for sure specify the full list of option she wants.
```
--cmake_extra_defines onnxruntime_WEBASSEMBLY_DEFAULT_EXTENSION_FLAGS=ON --cmake_extra_defines OCOS_ENABLE_OPENCV=ON``` 
will not work I think. Let me see if we can make it work.

File: cmake/external/extensions.cmake [hunk 1]
```diff
@@ -21,6 +21,21 @@ if (onnxruntime_REDUCED_OPS_BUILD)
...
+  set(OCOS_ENABLE_SPM_TOKENIZER ON CACHE INTERNAL "")
+  set(OCOS_ENABLE_GPT2_TOKENIZER ON CACHE INTERNAL "")
+  set(OCOS_ENABLE_WORDPIECE_TOKENIZER ON CACHE INTERNAL "")
+  set(OCOS_ENABLE_BERT_TOKENIZER ON CACHE INTERNAL "")
+  set(OCOS_ENABLE_TF_STRING ON CACHE INTERNAL "")
+  set(SPM_USE_BUILTIN_PROTOBUF OFF CACHE INTERNAL "")
+  set(OCOS_ENABLE_STATIC_LIB ON CACHE INTERNAL "")
+  set(OCOS_ENABLE_BLINGFIRE OFF CACHE INTERNAL "")
+  set(OCOS_ENABLE_OPENCV OFF CACHE INTERNAL "")
+  set(OCOS_ENABLE_CV2 OFF CACHE INTERNAL "")
```
Comment (wenbingl) [inline cmake/external/extensions.cmake:34]: _CV2 Replaced _OpenCV, did you intentionally to keep both of them for multiple versions?

Comment (guschmue) [reply cmake/external/extensions.cmake:34]: I saw both in the extension cmake so I thought play safe :).
I can remove _OpenCV

**Review by mszhanyi — APPROVED** [review]

**Review by mszhanyi — APPROVED** [review]

**Review by fs-eire — APPROVED** [review]

**Pull request merged**